openapi/components/schemas/ReportJournal.yaml

```
@@ -0,0 +1,54 @@
+type: object
+properties:
+  aggregationField:
+    type: string
+    description: The report data is grouped by the aggregation field.
+    enum:
+      - product.accountingCode
+      - product.id
+      - plan.id
+  currency:
+    $ref: CurrencyCode.yaml
+  bookedFrom:
+    type: string
+    description: A year and month in `yyyy-mm` format.
+    pattern: '^\d{4}-\d{2}$'
+    example: 2022-01
+  bookedTo:
+    type: string
+    description: A year and month in `yyyy-mm` format.
```

Suggestion based on `openapi/paths/reports@journal.yaml`

```suggestion
    description: Month in which revenue is booked until. Omit this value to retrieve booked revenue up until the most recently booked amount.
```